### ISSUE TITLE
Show the estimated print time and filament required for each model

### ISSUE THREAD

**Issue by ns-mglaske** [manyfold3d/manyfold#2039]
**Is your feature request related to a problem? Please describe.**
It would be nice to show the estimated print time, and filament required for each model.

**Describe the solution you'd like**
1) Add settings for printers, maybe take the printer definition(s) from Cura?
2) Show the estimated print time and filament required.

**Comment by matthewbadeau** [comment]
I think this is blocked until https://github.com/manyfold3d/manyfold/issues/3330 is completed. Without a built-in slicer, Manyfold won't be able to estimate how long a print will take for most model files.

There are hints in gcode that can help but this depends on gcode to be included with a model. Also the gcode may not even have all model files included with that specific gcode. So it makes this feature difficult.

Here is what the gcode hints look like from PrusaSlicer:
```
; filament used [mm] = 3285.98
; filament used [cm3] = 7.90
; filament used [g] = 9.80
; filament cost = 0.25
; total filament used [g] = 9.80
; total filament cost = 0.25
; total filament used for wipe tower [g] = 0.00
; estimated printing time (normal mode) = 35m 32s
; estimated printing time (silent mode) = 41m 17s
; estimated first layer printing time (normal mode) = 1m 12s
; estimated first layer printing time (silent mode) = 1m 17s
```

These hints are generated by the slicer and are just comments. So not all slicers even output these hints.

**Comment by Floppy** [comment]
Yeah, this issue should only cover details from existing gcode files - generating new ones depends on a lot of other stuff happening first.
